deploy/role.yaml

```diff
@@ -5,23 +5,25 @@ metadata:
 rules:
 - apiGroups:
   - kubevirt.io
-  - oauth.openshift.io
   - template.openshift.io
   resources:
   - '*'
   verbs:
-  - '*'
+  - create 
+  - get
+  - list
+  - patch
+  - update
+  - watch
+
```

Comment:
For (my) information: isn't this the whole list implied by '*'? Better explicit than implicit, though.

Reply:
'*' should contain three more actions: delete, deletecollection, proxy (i am not sure about this one)